src/components/study-container.jsx

```diff
@@ -313,6 +313,33 @@ export function StudyContainer({ view, onChangeTab }) {
         [snackError, userName]
     );
 
+    const sendAlert = useCallback(
+        (eventData) => {
+            const userId = eventData.headers[USER_HEADER];
+            if (userId !== userName) {
+                return;
+            }
+            const payload = JSON.parse(eventData.payload);
```

Comment:
Same, it is already parsed before calling the method, so it shouldn't be necessary to parse it again

Reply:
Without this parse, the payload is not properly handled, because the property names are still double quoted ...

Reply:
ok